src/DB_Command.php

```diff
@@ -650,6 +650,24 @@ public function size( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Display the database table prefix.
+	 *
+	 * Display the database table prefix specified in wp-config.php.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     $ wp db prefix
+	 *     wp_
+	 */
+	public function prefix() {
+		@WP_CLI::get_runner()->load_wordpress();
+
+		global $table_prefix;
+
+		WP_CLI::log( $table_prefix );
```

Comment:
Sorry, this should actually be `$wpdb->prefix`, so that it's set to the expected site on multisite.